app/models/project.rb

```diff
@@ -227,6 +227,12 @@ def inactive
     team.inactive
   end
 
+  def smooch_user_fields
```

Comment:
Same thing as I said above: let's try to avoid specific methods for annotation types or field names... we can have a method similar to `ProjectMedia`'s `get_annotations`, e.g., a generic method where we can pass the field name and/or annotation type as parameters.